src/comp/comp.py

```diff
@@ -24,48 +25,48 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [p.name for p in humans if p.name[0] == 'D']
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [p.name for p in humans if p.name[-1] == 'e']
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+c = [p.name for p in humans if p.name[0] in ['C','D','F','G']]
```

Comment:
There is an error on this line that is preventing one of the unit test for comp from passing.